node/Cargo.toml

```diff
@@ -56,7 +56,7 @@ num-rational = { version = "0.4.0", features = [ "serde" ] }
 num-traits = "0.2.10"
 num_cpus = "1"
 once_cell = "1"
-openssl = "0.10.32"
+openssl = "0.10.55"
```

Comment:
Is this necessary in this PR 🤔

Reply:
I'm OK in having it as a separate PR. I only put it here because we need to update the dependency for the CI to pass.

Reply:
Reverted this change for now. We can open a new PR with this or just merge back `dev` to this branch since the change is already there. WDYT?

Reply:
I think we should merge `dev` to `feat-1.6`.